### ISSUE TITLE
The parameter is incorrect.

### ISSUE THREAD

**Issue by OldScotsman** [rmcrackan/Libation#490]
I cannot change the location of the book storage, permanently.  When I change it back to what is has always been `D:\__AUDIOBOOKS\_____AUDIOBOOK CONVERSIONS\______LIBATION AUDIBLE DOWNLOAD MANAGER`
from the the one below which causes the error messages
`\\?\D:\__AUDIOBOOKS\_____AUDIOBOOK CONVERSIONS\______LIBATION AUDIBLE DOWNLOAD MANAGER`
and then save it, I find it just refers back to `\\?\` version when I log back into the settings.
I have submitted the last log  
Can you help?  
OldScotsman

**Log Files**
Attach your Libation log file here.
[Log202302_001.log](https://github.com/rmcrackan/Libation/files/10721825/Log202302_001.log)

**Comment by rmcrackan** [comment]
@Mbucari -- is this just a matter of settings showing the incorrect version of long path in the UI? The part about 'causes error messages' concerns me because it looks like a valid path.

**Comment by Mbucari** [comment]
@rmcrackan I guess the Windows GUI doesn't support the \\\\?\\ prefix. This must be the same underlying issue that caused #472. I've fixed it.

**Comment by OldScotsman** [comment]
Hello Mbucari
I have gone back in to Libation and it is still resetting the location of the"Book Storage" to a prefix of "\\?\" which is nor recognised.  The fix you mentioned, will this be part of an upgrade?
Regards
Bill

**Comment by Mbucari** [comment]
> The fix you mentioned, will this be part of an upgrade?

Yes, and it's still a work in progress because there are a couple of other issues being worked on right now.

Just so you know, the '\\\\?\\' prefix is used by windows to allow you to access files and folders whos path is longer than 260 characters. If you're happy with your books being stored at `D:\__AUDIOBOOKS\_____AUDIOBOOK CONVERSIONS\______LIBATION AUDIBLE DOWNLOAD MANAGER`, just leave it set to `\\?\D:\__AUDIOBOOKS\_____AUDIOBOOK CONVERSIONS\______LIBATION AUDIBLE DOWNLOAD MANAGER` and your books will end up in the right place.

**Comment by OldScotsman** [comment]
OK
Thank you.  You were most helpful and enlightening.
Regards
Bill
